src/documents/css/idg.css.styl

```diff
@@ -425,17 +425,23 @@ contrast-theme(foregroundColor, backgroundColor) {
     }
 
     .docs-template-category-active {
+        
         span {
             border-color: backgroundColor !important;
             background-color: backgroundColor !important;
             color: foregroundColor !important;
             border-bottom: 0.2rem solid backgroundColor !important;
         }
+        
         svg {
             fill: foregroundColor;
             background-color: backgroundColor !important;
         }
     }
+    
+    .docs-template-topics-active {
+          border-left: 0.3rem solid foregroundColor; 
```

Comment:
You can take this line (#422) and move it to line 428. You can then erase lines 442 and 444.